Builds/linux/README.md

```diff
@@ -6,10 +6,8 @@ builds, including docker based builds for those distributions, please consult
 the [rippled-package-builder](https://github.com/ripple/rippled-package-builder)
```

Comment:
This second line is irrelevant. This functionality has been subsumed by rippled the repo. (and I have a pull request the removes it again 🙄 )

Reply:
"number of processors" lines 41,152,173,183, referenced several times could be:
`$(echo $(nproc)-2 | bc)`
line 103 says `my_build` which won't be `.gitignore`d
I guess I can't comment on stuff you haven't changed.

Reply:
I would be glad to make those changes (and I might even have time to do it on a cleanly forked repo, so crufty PRs don't make their way in).

However, it will be Thursday at the earliest, since I'm working tonight.